index.html

```diff
@@ -233,21 +409,22 @@ <h1>About me</h3>
             <div class="form-group">
                 <input class="input" type="text" name="name" maxlength="30" placeholder="Full name" rows="10" required>
                 <input class="input" type="email" name="email" placeholder="Email address" required>
-                <textarea name="send" maxlength="500" required placeholder="Write me something"></textarea>
+                <textarea name="send" maxlength="399" required>Write me something</textarea>
                 <input class="submit" type="submit" value="Get in touch">
             </div>
         </form>
     </section>
-    <hr>
+    <hr class="hr">
     <footer class="footer">
         <div class="icon">
             <ul>
                 <li><a href="https://github.com/BlessmanNewton"><i class="fa fa-github" aria-hidden="true"></i></a></li>
-                <li><a href="https://linkedin.com/BlessmanNewton"><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
-                <li><a href="https://angellist.com/BlessmanNewton"><i class="fa fa-angellist" aria-hidden="true"></i></a>
-                </li>
-                <li><a href="https://twitter.com/BlessmanNewton"><i class="fa fa-twitter" aria-hidden="true"></i></i></a>
+                <li><a href="https://linkedin.com/BlessmanNewton"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
                 </li>
+                <li><a href="https://angellist.com/BlessmanNewton"><i class="fa fa-angellist"
+                            aria-hidden="true"></i></a></li>
+                <li><a href="https://twitter.com/BlessmanNewton"><i class="fa fa-twitter"
+                            aria-hidden="true"></i></i></a></li>
                 <li><a href="https://medium.com/BlessmanNewton"><i class="fa fa-medium" aria-hidden="true"></i></a></li>
             </ul>
         </div>
```

Comment:
Kindly increase the margin-bottom after the `Get in touch` button to have your design similar to the figma design :

Figma design :
![image](https://github.com/Blessman-Newton/Portfolio/assets/20537712/a15615d5-5851-4fb6-804d-351f8cd2430c)

Your design :
![image](https://github.com/Blessman-Newton/Portfolio/assets/20537712/0a934a20-9d86-4b25-8a5a-63764e7f79a1)